kafka-client/src/main/java/dev/responsive/kafka/config/ResponsiveDriverConfig.java

```diff
@@ -64,6 +64,12 @@ public class ResponsiveDriverConfig extends AbstractConfig {
       + "responsive server. This applies both to metadata requests and query execution.";
   private static final long REQUEST_TIMEOUT_MS_DEFAULT = 5000L;
 
+  // ------------------- other configurations ---------------------------------
+  public static final String PARTITION_EXPLODE_FACTOR_CONFIG = "responsive.partition"
+      + ".explode.factor";
+  public static final String PARTITION_EXPLODE_FACTOR_DOC = "Explodes kafka partitions "
+      + "into remote partition counts";
+  public static final int PARTITION_EXPLODE_FACTOR_DEFAULT = 31;
```

Comment:
why 31?

Reply:
ah, originally I was using an algorithm that wouldn't work if you exploded by the same number as partitions so I wanted to choose an unlikely number. that's no longer a limitation. Maybe I'll change this config actually so that it'll be a target number of partitions instead of a factor.